ci/azure-linux_mac.yml.diff

```diff
@@ -0,0 +1,162 @@
+diff --git a/ci/azure-linux_mac.yml b/ci/azure-linux_mac.yml
```

Comment:
Do we want this diff file?

Reply:
Oh that was committed accidentally. I will remove it.